src/browser/index.ts

```diff
@@ -1,17 +1,16 @@
+import EventSource from 'eventsource'
```

Comment:
Importing this module will increase the bundle size quite a bit.

After discussing this internally, I think what we should do something like this:

* Add `EventSource` as an optional `Config` field
* Leave `window.EventSource` as the default
* Check `if (config.token && config.EventSource === DEFAULT_EVENT_SOURCE)` -> throw like it currently is doing

This way we:
* Maintain 100% backwards compatibility with existing installs
* Allow you to provide whatever eventsource library you deem fit for your usecase.
** `'eventsource'` dependency will add 200kb+ to your bundle; consider using `@sanity/eventsource` which uses `eventsource` in node and `event-source-polyfill` in the browser.

Does this make sense?

Reply:
Hello again! I'm thinking, to avoid installing another library in the code using this library (i.e. installing `eventsource` in `next-sanity`), to have a string rather than an event source as input argument (maybe this is what you mean anyway?)

i.e. something like

```typescript
# types.ts

type BrowserEventstreamType = "default" | "polyfill"

export interface Config {
  ...
  subscriptionThrottleMs?: number
  browserEventStreamType?: BrowserEventstreamType
}
```

Not very happy with the name `BrowserEventstreamType` though 🤔

What do you think?

Reply:
Something like this? https://github.com/sanity-io/groq-store/pull/9/commits/650b937fe50b4ddf91fc9b1baf96bfaac49759e3

Will update https://github.com/sanity-io/next-sanity/pull/51 accordingly once this is ready